python/mxnet/executor.py

```diff
@@ -26,7 +26,15 @@ def callback_handle(name, array, _):
     return callback_handle
 
 class Executor(object):
-    """Executor is the actual executing object of MXNet."""
+    """Executor is the actual executing object of MXNet,
```

Comment:
This is a circular definition "Executor is the actual executing object". Can you replace with a more descriptive explanation of what precisely the executor does?